SyncUp.Web/src/components/JoinButton.vue

```diff
@@ -1,6 +1,5 @@
 <template>
   <v-btn
-    width="100px"
```

Comment:
This was here to prevent UI jumping when it switches between the words join and joined